stories/animation/doc-elements.tsx

```diff
@@ -21,7 +22,7 @@ export const AnimationPane = styled.div`
     display: flex;
     justify-content: center;
     align-items: center;
-    background: ${Color.Accent.Light[6]};
+    background: #fbfcfe;
```

Comment:
Change this to keep constant. Else could change if we change the theme in storybook